googlepubsub/json_schemas/channel.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
in both schemas, you're missing `bindingVersion` property. Please have a look at schemas from other bindings. You also do not have it in markdown definitions. It is necessary for user to have it, so they can specify what version of binding they're using. Bindings specs are versioned independently from main spec

Reply:
My apologies, I'll update the PR to have these in both markdown and schemas.